spec/index.html

```diff
@@ -954,10 +1060,21 @@ <h2>Annotation Syntax</h2>
         and the RDF Abstract Syntax [[RDF11-CONCEPTS]] does not
         distinguish how the triples were written.</p>
 
+      <p class="note">When using RDF 1.2-specific features, such as
```

Comment:
This seems to be duplicate text to lines 967-974.

Reply:
+1

Its copied verbatim so the annotation syntax refers to triple terms which aren't visible.

Reply:
Now it should only appear in the "Version Announcement" section.